book/src/configuration/proxy.md

```diff
@@ -6,47 +6,29 @@ Proxy settings for Halloy.
 
 ```toml
 [proxy]
-type = "socks5"
-host = "192.168.1.100"
-port = 1080
+Socks5 = { host = "192.168.1.100", port = 1080 }
```

Comment:
How does toml handle a unit type for `Tor` variant? 

```toml
[proxy]
Tor
```

Doesn't seem like valid toml but I'm on my phone and can't validate. Or do we need:

```toml
[proxy]
Tor = {}
```

Might be good to document

Reply:
The latter one is correct.